README.md

```diff
@@ -15,6 +15,13 @@ Other resources: [consul.io/community](https://www.consul.io/community)
 
 Additionally, for issues and pull requests, we'll be using the 👍 reactions as a rough voting system to help gauge community priorities. So please add 👍 to any issue or pull request you'd like to see worked on. Thanks.
 
+## Consul-Terraform-Sync Roadmap
+
+Knowing about our upcoming features and priorities helps our users plan. This repository contains information about what we are working on and allows all our user to give direct feedback.
+
+
+[See the roadmap »](https://github.com/hashicorp/consul-terraform-sync/projects/1)
+
```

Comment:
There's a bit of a context change between this section and the FAQs at the end of the file, maybe add another link to the FAQ somewhere here could tie them together a bit better

Reply:
Adding this statement to tie FAQ together.
"Please visit the Roadmap FAQs for more information on the categorization of the roadmap"

Reply:
Added this statement to link FAQ with roadmap
"Please visit the Roadmap FAQs for more information on the categorization of the roadmap"